ts/RailroadMap.ts

```diff
@@ -760,11 +763,25 @@ export class RailroadMap {
 
     private renderBorder(): Element {
         // Border
-        return this.layers.border
-            .rect(4_000_00, 4_000_00)
-            .translate(-2_000_00, -2_000_00)
-            .radius(100_00)
-            .addClass('map-border');
+        if (this.railroad.settings.gameLevelName === 'LakeValley') {
+            return this.layers.border
+                .rect(4_000_00, 4_000_00)
+                .translate(-2_000_00, -2_000_00)
+                .radius(100_00)
+                .addClass('map-border');
+        } else if (this.railroad.settings.gameLevelName === 'AuroraFalls') {
+            return this.layers.border
+                .rect(6_000_00, 6_000_00)
+                .translate(-3_000_00, -3_000_00)
+                .radius(100_00)
+                .addClass('map-border');
+        } else (this.railroad.settings.gameLevelName === 'PineValley'); {
```

Comment:
Pine valley maps have gameLevelName set to null.

```suggestion
        } else {
```